projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -256,6 +256,20 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "// TODO:Spartacus - Property 'authRedirectService' was removed from Class 'CdcLogoutGuard'. It is not used anymore.  It was there only to pass to the super() contructor and the super() constructor had this parameter removed.",
     },
+    {
+      class: 'CdsMerchandisingProductService',
+      importPath: '@spartacus/cds',
+      deprecatedNode: 'loadProductsForStrategy',
+      comment:
+        "// TODO:Spartacus - The 'loadProductsForStrategy' method's signature changed to: 'loadProductsForStrategy(  strategyId: string,  numberToDisplay: number): Observable<StrategyResponse>'",
+    },
+    {
+      class: 'CdsMerchandisingProductService',
+      importPath: '@spartacus/cds',
+      deprecatedNode: 'merchandisingSearchContextService',
```

Comment:
`merchandisingSearchContextService` was removed from constructor.

Reply:
The constructor change entry is already present in `generated-constructor.migration.ts` for `CdsMerchandisingProductService`.  When a constructor argument is removed, it is expected to have entries in `generated-constructor.migration.ts` to register the constructor change.  

At the same time, this also removes a property from the class. Therefore there is also an entry in `generated-methods-and-properties.migration.ts`.